packages/tools/devtools/devtools-browser-extension/e2e-tests/app/app.tsx

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import { CollaborativeTextArea, SharedStringHelper } from "@fluid-experimental/react-inputs";
+import { DevtoolsLogger, IDevtools, initializeDevtools } from "@fluid-experimental/devtools";
+import { IFluidContainer, ContainerSchema } from "@fluidframework/fluid-static";
+import { SharedMap } from "@fluidframework/map";
+import { TinyliciousClient } from "@fluidframework/tinylicious-client";
+import { SharedString } from "@fluidframework/sequence";
+import { ContainerKey } from "@fluid-experimental/devtools";
```

Comment:
I'm surprised CI didn't catch this "duplicate" import, we already have an entry to import from this package above in line 9. Let's consolidate this import into that line too.